### PR TITLE
Add Dockerfile for easy local testing

### PR DESCRIPTION
**Pull request by rawbertp**
I created a `Dockerfile` which allows for easy local testing. This is for development purposes only and might be useful for other contributors too.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+# (1) Build image (only required once or after this file was changed):
+#     $ docker build -t photoboothbase .
+# (2) Start the container:
+#     $ docker run --rm -v "$(pwd):/var/www/html/" -p 8080:80 photoboothbase
+# (3) Go to: http://localhost:8080
+
+RUN set -x \
+    && apt-get update && apt-get dist-upgrade -y \
+    && apt-get dist-upgrade -y \
+    && apt-get install -y nginx php-fpm gphoto2 unclutter
```
Comment (andi34) [inline Dockerfile]: php-fpm includes gd? Or we also need php-gd?

**Comment by andi34** [comment]
"apt install" should be enough instead "apt-get install"

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+# Usage:
+# (1) Build image (only required once or after this file was changed):
+#     $ docker build -t photoboothbase .
+# (2) Start the container:
+#     $ docker run --rm -v "$(pwd):/var/www/html/" -p 8080:80 photoboothbase
+# (3) Go to: http://localhost:8080
+
+RUN set -x \
+    && apt-get update && apt-get dist-upgrade -y \
+    && apt-get dist-upgrade -y \
```
Comment (andi34) [inline Dockerfile]: dist-upgrade twice?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    && apt-get install -y nginx php-fpm gphoto2 unclutter
+
+RUN set -x \
+    && apt-get update \
+    && apt-get install -y curl vim gnupg git
```
[the remark this answers is not in the document]
Comment (rawbertp) [reply Dockerfile]: - As this is for dev purposes only, `vim` comes in handy when you connect to the container and want to quickly edit stuff.
- `curl` and `gnupg` and `git` are required for the `yarn` installation.

File: Dockerfile (hunk 1 above)
Comment (rawbertp) [reply Dockerfile]: You're right - that was missing. Added it.

File: Dockerfile (hunk 2 above)
Comment (rawbertp) [reply Dockerfile]: Thanks, removed the second one.

**Comment by rawbertp** [comment]
> "apt install" should be enough instead "apt-get install"

See here why `apt-get` should be used in scripts rather than `apt`: https://askubuntu.com/a/990838

**Comment by andi34** [comment]
> > "apt install" should be enough instead "apt-get install"
> 
> See here why `apt-get` should be used in scripts rather than `apt`: https://askubuntu.com/a/990838

Then Line 30 (`&& apt update && apt install -y yarn`) needs an update

**Comment by sualko** [comment]
Looks good, but I have to give it a try if I have more time.

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+#     $ docker run --name photobooth --rm -v "$(pwd):/var/www/html/" -p 8080:80 photobooth
+# (3) Go to: http://localhost:8080
+#
+# If you would like to connect to the running container:
+# $ docker exec -it photobooth /bin/bash
+
+RUN set -x \
+    && apt-get update \
+    && apt-get dist-upgrade -y \
+    && apt-get install -y nginx php-fpm php-gd gphoto2 unclutter
```
Comment (andi34) [inline Dockerfile]: unclutter is not needed here as we don't enable kiosk mode

Comment (rawbertp) [reply Dockerfile]: Right... removed it.

**Pull request merged**